### PR TITLE
release(JsAst) coq-jsast version 2.0.0

### PR DESCRIPTION
**Pull request by jeromesimeon**
Signed-off-by: Jerome Simeon <jeromesimeon@me.com>

- Supports Coq 8.11.2 and 8.12.0
- Switches from Flocq to native Coq floats

### CODE REVIEW

File: released/packages/coq-jsast/coq-jsast.2.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+bug-reports: "https://github.com/querycert/jsast/issues"
+dev-repo: "git+https://github.com/querycert/jsast/tree/JsAst"
+
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/JsAst"]
```
Comment (palmskog) [inline released/packages/coq-jsast/coq-jsast.2.0.0/opam:37]:
```suggestion
```
Redundant line.

File: released/packages/coq-jsast/coq-jsast.2.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+name: "coq-jsast"
+version: "2.0.0"
```
Comment (palmskog) [inline released/packages/coq-jsast/coq-jsast.2.0.0/opam:3]:
```suggestion
```
Name and version are already given by opam file path.

File: released/packages/coq-jsast/coq-jsast.2.0.0/opam [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/JsAst"]
+depends: [
+  "ocaml" {>= "4.08.0"}
```
Comment (palmskog) [inline released/packages/coq-jsast/coq-jsast.2.0.0/opam:39]: This is a pure Coq development, right? If so, why do you need this bound on OCaml?

File: released/packages/coq-jsast/coq-jsast.2.0.0/opam (hunk 2 above)
Comment (jeromesimeon) [reply released/packages/coq-jsast/coq-jsast.2.0.0/opam]: Removed.

File: released/packages/coq-jsast/coq-jsast.2.0.0/opam (hunk 1 above)
Comment (jeromesimeon) [reply released/packages/coq-jsast/coq-jsast.2.0.0/opam]: Removed that line, but I'm unclear about the rules here. We had a similar line in this package: https://github.com/querycert/opam-coq-archive/blob/06f9f294f3ae8378b4dfbdaade628c123054ac20/released/packages/coq-qcert/coq-qcert.2.0.0/opam#L25

Is that also redundant?

File: released/packages/coq-jsast/coq-jsast.2.0.0/opam (hunk 3 above)
Comment (jeromesimeon) [reply released/packages/coq-jsast/coq-jsast.2.0.0/opam]: No good reason at all. Removed.

File: released/packages/coq-jsast/coq-jsast.2.0.0/opam (hunk 1 above)
Comment (clarus) [reply released/packages/coq-jsast/coq-jsast.2.0.0/opam]: Yes, opam 2 handles the removal automatically by tracking the installed files. This did not exist with opam 1.

Comment (jeromesimeon) [reply released/packages/coq-jsast/coq-jsast.2.0.0/opam]: > Yes, opam 2 handles the removal automatically by tracking the installed files. This did not exist with opam 1.

Thanks for clarifying. Unless you want me to I won't change the `coq-qcert` opam file, but will simply remove that line in future versions.

**Pull request merged**